Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -330,7 +328,6 @@ def serialized_identifiers():
         'WebKit::WebGPUIdentifier',
         'WebKit::WebPageProxyIdentifier',
         'WebKit::WebURLSchemeHandlerIdentifier',
-        'WebKit::XRDeviceIdentifier',
     ]
```

Comment:
Alex can comment more, but the condition for the identifier could be added to the hardcoded list. E.g.
`('WebKit::XRDeviceIdentifier, 'ENABLE(WEBXR)'),`

and then the generator iteration part can do
```
if isinstance(I, str):
    ....
else # assume tuple (name, condition)
```

Reply:
Seems straightforward enough. I can implement this if you'd like.